polaris-react/src/components/CalloutCard/CalloutCard.tsx

```diff
@@ -40,7 +40,7 @@ export function CalloutCard({
   const primaryActionMarkup = buttonFrom(primaryAction);
   const secondaryActionMarkup = secondaryAction
     ? buttonFrom(secondaryAction, {
-        variant: 'tertiary',
+        variant: secondaryAction.variant || 'tertiary',
```

Comment:
Shouldn't matter in this case but we use nullish coalescing for this type of thing

```suggestion
        variant: secondaryAction.variant ?? 'tertiary',
```